sos/report/plugins/monit.py

```diff
@@ -21,7 +21,6 @@ class Monit(Plugin, RedHatPlugin):
     plugin_name = 'monit'
 
     # Define configuration files
-    # FIXME: direct globs will fail in container environments.
```

Comment:
Pylint doesn't like FIXME comments. This was added in 2016. If no one's affected yet, I thought we can ignore it until someone actually hits this issue.

The alternative is to create an issue and link it there and say 'it's tracked in #issue'. Let me know if this is preferable here.

Reply:
ACK to remove this. If nobody has complained in past 8 years, most probably nothing to fix.